src/Avalonia.Base/Input/GestureRecognizers/ScrollGestureRecognizer.cs

```diff
@@ -111,21 +113,25 @@ public void PointerMoved(PointerEventArgs e)
                         _scrolling = true;
                     if (_scrolling)
                     {
+                        _velocityTracker = new VelocityTracker(); // TODO: Should be platform specific -- this default tracker is for Android.
```

Comment:
I think these comments can be removed. It's not much "TODO" as a just a possibility to add more trackers.